pyproject.toml

```diff
@@ -11,6 +11,8 @@ readme = "README.md"
 license = "Apache-2.0"
 keywords = ["Kolena", "ML", "testing"]
 classifiers = [# classifiers for license, versions set automatically during Poetry build
+    "Programming Language :: Python :: 3",
```

Comment:
Let's add specific version tags as well, for each supported Python version

Reply:
Updated to include the supported versions and removed poetry line